signer/src/storage/postgres.rs

```diff
@@ -361,10 +396,74 @@ impl super::DbRead for PgStore {
 
     async fn get_pending_withdraw_requests(
         &self,
-        _chain_tip: &model::BitcoinBlockHash,
-        _context_window: usize,
+        chain_tip: &model::BitcoinBlockHash,
```

Comment:
Why take this as input? Don't we always want to fetch pending withdrawals referenced from the bitcoin `chain_tip`?

Reply:
I think it makes the behavior easier to reason about since we can control so that the transaction has the same bitcoin chain tip across iterations. Otherwise we could make a query for one chain tip, and then the chain tip changes before the next query. I'm not sure it would cause any problems, but it feels simpler to be able to control it.

Reply:
Ahh I see, yeah that makes sense.

Reply:
Also I realized the coordinator will want to fetch withdraw requests using a context window starting 6 blocks from the chain tip, so there is a use case for not always fetching the latest as well :)